### PR TITLE
modify  wf actions `.yml`

### PR DESCRIPTION
**Pull request by yibeichan**
1. switch from setup-python to setup-miniconda 
> `datalad` needs `git-annex`, which can only be installed through `conda` not `pip`
2. add `etc/environment.yml` for conda env `pydra-tutorial`
3. remove `requirements.txt`

### CODE REVIEW

**Comment by yibeichan** [comment]
@effigies sorry I didn't know that even if I switch to a new branch, all commits I have from different branches will still appear in one place. Would you like to review the two `.yml` files I modified?

**Comment by djarecka** [comment]
@yibeichan - this PR contain all the changes rom #34, am I right?

**Comment by yibeichan** [comment]
it looks so. Here it shows 24 commits, but no testing is going on. (I'm a bit confused about how different branches work here

File: .github/workflows/notebooks.yml [hunk 1]
```diff
@@ -20,11 +20,13 @@ jobs:
     - name: Disable etelemetry
       run:  echo "NO_ET=TRUE" >> $GITHUB_ENV
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
+    - uses: conda-incubator/setup-miniconda@v2
```
Comment (effigies) [inline .github/workflows/notebooks.yml:23]:
```suggestion
      uses: conda-incubator/setup-miniconda@v2
```

File: .github/workflows/gh-pages.yml [hunk 2]
```diff
@@ -12,16 +12,18 @@ jobs:
...
     - name: Disable etelemetry
       run: echo "NO_ET=TRUE" >> $GITHUB_ENV
+    - name: Active conda env pydra-tutorial
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        activate-environment: pydra-tutorial
+        environment-file: etc/environment.yml
+        python-version: ${{ matrix.python-version }}
+        auto-activate-base: false
+      run: conda activate pydra-tutorial
```
Comment (effigies) [inline .github/workflows/gh-pages.yml:24]:
```suggestion
```

**Review by effigies — COMMENTED** [review]
A couple syntax errors.

File: .github/workflows/notebooks.yml [hunk 3]
```diff
@@ -20,11 +20,13 @@ jobs:
...
-      uses: actions/setup-python@v2
+      uses: conda-incubator/setup-miniconda@v2
       with:
+        activate-environment: pydra-tutorial
+        environment-file: etc/environment.yml
         python-version: ${{ matrix.python-version }}
-    - name: Install requirements
-      run: python -m pip install -r requirements.txt
+        auto-activate-base: false
+      run: conda activate pydra-tutorial
```
Comment (effigies) [inline .github/workflows/notebooks.yml:29]:
```suggestion
```

**Comment by yibeichan** [comment]
This one was fixed in #36 so we can close it

**Pull request closed without merging**